docs/provider/provider-tls.md

```diff
@@ -162,9 +162,9 @@ server {
 </TabItem>
 </Tabs>
 
-:::tip
+:::caution
 
-The above examples use ports `2223` and `2224`, respectively. You can choose any port that works for your purposes. Be aware that some ports on your OS may be used for internal communication and should be avoided. 
+The above examples use ports `443` for external listening and `2223` / `2224` for internal comms, respectively. Using ports other than `443` for external listening means that some users will not be able to connect to your provider. This can result in less rewards and poorer quality of service. For internal listening, be aware that some ports on your OS may be used for internal communication and should be avoided. 
```

Comment:
Should the ports 2223 and 2224 be public accessible? If not, could you mention to close those ports with a firewall rule?

Reply:
I don't think they need to be, but I'd need to @ranlavanet to weigh in to be definite about it... He's far more knowledgeable about what nginx actually does with this config than I am.